internal/pkg/instrumentation/version.go

```diff
@@ -0,0 +1,13 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Package instrumentation provides functionality to manage instrumentation
+// using eBPF for Go programs.
+package instrumentation
+
+const (
+	// DistributionName is used for `telemetry.distro.name` resource attribute.
+	DistributionName = "opentelemetry-go-instrumentation"
+	// DistributionVersion is the current release version of OpenTelemetry Go auto-instrumentation in use.
+	DistributionVersion = "v0.22.1"
```

Comment:
What do you think about dropping the "Distribution" part of the const names?


```suggestion
	// Name is used for `telemetry.distro.name` resource attribute.
	Name = "opentelemetry-go-instrumentation"
	// Version is the current release version of OpenTelemetry Go auto-instrumentation in use.
	Version = "v0.22.1"
```

This will change use from `instrumentation.DistributionName` and `instrumentation.DistributionVersion` to `instrumentation.Name` and `instrumentation.Version` which read a bit better and are shorter to type.